packages/trader/src/Modules/Trading/Containers/trade-params.jsx

```diff
@@ -19,16 +20,20 @@ const TradeParams = ({ form_components, is_minimized }) => {
     const isVisible = component_key => {
         return form_components.includes(component_key);
     };
+
     return (
         <React.Fragment>
             {isVisible('duration') && <Duration key={'duration'} is_minimized={is_minimized} />}
             {isVisible('barrier') && <Barrier key={'barrier'} is_minimized={is_minimized} />}
+            {isVisible('barrier_selector') && <BarrierSelector key={'barrier_selector'} />}
             {isVisible('last_digit') && <LastDigit key={'last_digit'} is_minimized={is_minimized} />}
-            <Fieldset className={classNames('trade-container__fieldset', 'trade-container__fieldset--no-padding')}>
-                {isVisible('vanilla_trade_type') && <VanillaTradeTypes key={'vanilla_trade_type'} />}
-                {isVisible('strike') && <Strike key={'strike'} />}
-                {isVisible('amount') && <Amount key={'amount'} is_minimized={is_minimized} />}
-            </Fieldset>
+            {(isVisible('vanilla_trade_type') || isVisible('strike')) && (
+                <Fieldset className={classNames('trade-container__fieldset', 'trade-container__fieldset--no-padding')}>
+                    {isVisible('vanilla_trade_type') && <VanillaTradeTypes key={'vanilla_trade_type'} />}
+                    {isVisible('strike') && <Strike key={'strike'} />}
+                </Fieldset>
+            )}
```

Comment:
Strike param should be separated from Stake (Amount) param in Vanillas, as requested by designers